### ISSUE TITLE
FXIOS-809 ⁃ Cannot generate l10n screenshots: no such module 'WidgetKit'

### ISSUE THREAD

**Issue by JohanLorenzo** [mozilla-mobile/firefox-ios#7220]
Yesterday @Delphine asked me to generate the screenshots. I've hit 2 compilation errors since then. The first one was caused by https://github.com/mozilla-mobile/firefox-ios/pull/7112. https://github.com/mozilla-mobile/firefox-ios/pull/7216 fixed it. The second error is explained below.

## Steps to reproduce

1. Trigger the l10n screenshot hook: https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-mobile-firefox-ios%2Fl10-screenshots.
2. Wait for the task to finish and look at its logs. You will get the taskId of the generated decision task. In my case: https://firefox-ci-tc.services.mozilla.com/tasks/Un6660b6S4Ojno7iEZrjTw/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FUn6660b6S4Ojno7iEZrjTw%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L96 points to https://firefox-ci-tc.services.mozilla.com/tasks/groups/G3o8KpATSwC-Y80bMB8ydw
3. See the failed `build-screenshots` task. It failed to import WidgetKit. https://firefox-ci-tc.services.mozilla.com/tasks/B0lC-cizRmC6JS3peUrAXQ/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FB0lC-cizRmC6JS3peUrAXQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Fbitrise.log#L7208

```
❌  /Users/vagrant/git/WidgetKit/SearchQuickLinksMedium/SearchQuickLinks.swift:5:8: no such module 'WidgetKit'
```

### Expected behavior
No compilation error.

### Actual behavior
See error above.

### Additional notes

I tried to `git bisect` to know what commit introduced this module, but I didn't anything. `git grep WidgetKit` doesn't return anything either. I don't have enough knowledge about iOS to know what's the right thing to do.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-809)

**Comment by garvankeeley** [comment]
I am trying to uplift patches from `iOS14` branch to `main` as we migrate `main` to that platform. I am blocked on the remaining uplift which would bring the project back to building, so I might have to back out the last WidgetKit patch if I can't fix this quickly. Of concern is that `main` is switching to Xcode Beta 5 and Bitrise doesn't support specific Xcode beta version picking, and likely is on Beta 6 on their builders.

**Comment by JohanLorenzo** [comment]
Thanks for the context @garvankeeley! I'm using Xcode release on my machine and I got the same error as the builder. So I guess the bug is somewhere else. 

I followed the instructions you gave me on Slack, and I triggered the screenshots against the `v28.1` branch, instead of the `main` one. The `build-screenshots` task passed! https://firefox-ci-tc.services.mozilla.com/tasks/groups/bwPMG4-IS6-BfwcwF7YFlw

I'm going to close this bug, because I guess it'll be fixed before we reach next release and I'll likely get screenshots this time.
